### ISSUE TITLE
EPIC: Update all SD ecosystem logos for 1.0

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-ux#72]
## Problem
With the 1.0 deployment of SecureDrop, a number of UI updates will also be pushed. The goal with these updates is bring more alignment to the SD brand, to promote trust and credibility in the product and in all product touchpoints.

## Solution
Identify all touchpoints across the SD ecosystem that have logos on them. In parallel to that, identify who the users are that use those touchpoints, and what their motivations are.

Then, craft updated lockups to push to each touchpoint, based on the below target.

Lastly, define a system of rules that all of these logo variations fit within, and publish that in the #65 and Brand Use Guidelines that @huertanix is working on. 

Stretch goal: Craft a page for the Admin Guide that defines governance of the Newsroom Logo w/in the app, and likely co-branding.

## Prioritization
When a product or service's brand-updates are published that do not involve a total-logo/identity redesign (such as Slack and AirBnB's logo updates from the past few years), it is typically _desired_ to push the updated logo at the same time the primary product pushes the update—but that is _rarely_ a `MUST` dependency. With a total redesign, it conversely IS a MUST dependency.

This endeavor is for an _update_ to the SecureDrop logo, not a redesign. As such, it will be _desired_ to update the logo across all touchpoints, but not a mandate. The milestone for this issue is therefore a `SHOULD` target, and not a `MUST`.

**Comment by ninavizz** [comment]
# Identified Touchpoints
## SD schwag
_**Users:**_ —
_**Motivations:**_ —
FPF Store: https://teespring.com/stores/freedom-of-the-press
Q for @zenmonkeykstop @huertanix: are there print materials we send to newsrooms today?
## Training Materials
@olivemartini @huertanix  halp, here? 😄 
## SD.org Website
**_Users:_** —
**_Motivations:_** —
GH Issue: #71 
## Social Media
- Twitter
- ??? 
## FPF Website
**_Users:_** —
**_Motivations:_** —
[ need imgs ]
## Read The Docs
**_Users:_** Primary, Newsroom folks; Secondary, contributors; Tertiary, maaaybe Sources
**_Motivations for use:_** Learn about the product
![image](https://user-images.githubusercontent.com/8262612/61158094-d8320e00-a4ac-11e9-8b61-45a249ea7276.png)
## Redmine
Users: Newsroom folks
Motivations: Connect w/ FPF peeps for help
[ need img ]
## Forums
**_Users:_** —
**_Motivations:_** —
## Localization Lab (?)
**_Users:_** —
**_Motivations:_** —

**Comment by ninavizz** [comment]
# Priority touchpoints for 1.0 release
_Discuss as team in UX mtng_

**Comment by eloquence** [comment]
Twitter is a high priority IMO; not necessarily for this sprint, but we'll want to line it up so we have it ready to go on release day (September 4). The old logo has been cut off since Twitter switched to round avatars. :/ The letterforms are also more pixelated than I think can be explained by Twitter's image compression:
https://twitter.com/SecureDrop/photo

**Comment by ninavizz** [comment]
### 08 Aug UX Mtng Notes
- Twitter = priority
   - No attachment to FPF banner image
      - Something more SD is desired
   - Lots of Erik `mmh` wrt how current lockup is cropped by Twitter's circle schtick
   - No discussion of logo vs lockup; Nina doesn't feel lockup is appropriate for the avatar, will create mox with both.
- RTD = nice-to-have
- Redmine is FPF, not SD branded
- Contributor sites/pages = low priority
- Schwag touched upon, but not discussed
   - `nina FFS` let's at least update the stickers w/ a logo-only diecut sticker, ad a mug with the lockup, and pull the existing SD tee. #micdrop

**Comment by eloquence** [comment]
> Redmine is FPF, not SD branded

Redmine does have a SecureDrop favicon; we'll need to decide which favicon variant to update it to (IMO the final monochrome variant we'll use for the website).

**Comment by eloquence** [comment]
Note: Per discussion w/ Nina, we'll want to be a bit more consistent about filling in the spaces within the logomark (the cube), instead of marking them transparent. The favicon has them white, but the version linked from the [branding guide](https://securedrop.org/brand-refresh-guide-2019/) uses transparency throughout.

**Comment by ninavizz** [comment]
^ Thought about this last night, and realized I'd done this entirely in error. The monochromatic versions were intended to be transparent, but not the Full-Color one. Will update FullColor art over the weekend, and post here for updating on GH at your soonest convenience, Erik!

**Comment by ninavizz** [comment]
^^
- **DONE!!** 
  - Individual `.ai` `SVG` and `PNG` files updated in `Brand Assets` directory w/in this repo
  - ZIP archive re-created & also updated in `Brand Assets` directory, with above cited updated files 

@eloquence Do the files need to be updated elsewhere for newsrooms to download the correct versions from the PDF, or does that happen via a URL forward from the `securedrop.org` link in the PDF?

**Comment by eloquence** [comment]
> Do the files need to be updated elsewhere for newsrooms to download the correct versions from the PDF

Nope, we're all set! The PDF points to the latest version of the ZIP file. Thanks very much for these quick fixes! :)

**Comment by ninavizz** [comment]
@huertanix Ping ping!! This is an Issue I created to function as a branding/logo-use inventory for all ecosystem touchpoints.

**Comment by ninavizz** [comment]
Training materials, general presentation/sales materials, fundraising stuff, Localization Lab, and forums »» Have these touchpoints been updated, yet? /cc @eloquence @rocodes 

Website is in a separate issue. Lookin' to close this one.

**Comment by eloquence** [comment]
Forums finally updated as of 12/6; thanks to @ninavizz for preparing assets for that :)

**Comment by ninavizz** [comment]
To close this issue (a few weeks after we've moved up to `2.0.1`, heh) I'd like to prioritize one more thing: including the current brand stuff in the docs, with some cleanup to the landing page recommendations. That is the lone reason I'm putting the "prioritize, yo!" tag on this, so that part can be prioritized.

**Comment by huertanix** [comment]
@ninavizz Just now seeing this ticket; Sorry for the delay! There are no printed materials we use for SD trainings. The Teespring store is being retired so any new merch should use the new blue logo. The slide deck template for SD trainings was updated to use the new logo and colors a year-ish(?) ago, although I think not too many people on the SD team know it exists yet.

**Comment by ninavizz** [comment]
Final outstanding need to close this issue: Concretely document brand use guidelines for updated SD stuff. Include CMYK color alternatives, and basic print design guidance for use of RGB vs CMYK vs Spot colors.
